doc/src/selections.rst

```diff
@@ -54,11 +53,36 @@ false and false to true. ``name(#1) == H and not x(#1) < 5.0`` and ``(z(#2) < 45
 and name(#4) == O) or name(#1) == C`` are complex selections using booleans
 operators.
 
-List of implemented properties
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+List of implemented selectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Here is the list of currently implemented properties (either string or numeric
-properties). Additional properties ideas are welcome!
+Here is the list of currently implemented selectors. Additional ideas are welcome!
+
+Boolean selectors
+-----------------
+
+- ``all``: always matches the atom and return ``true``;
+- ``none``: that matches no atom and always return ``false``;
+- ``bonded(i, j)``: check if atoms i and j are bonded together. If i and j
```

Comment:
Maybe rename is_bonded as to be consistent with the other functions?

Reply:
Done